### PR TITLE
Use -Wall in ros_control

### PR DESCRIPTION
**Pull request by jaagut**
## Proposed changes
This adds the -Wall -Werror -Wno-unused flags to ros_controll.

## Necessary checks
- [x] Run `catkin build`
~- [ ] Write documentation~
~- [ ] Create issues for future work~
~- [ ] Test on your machine~
- [ ] Test on the robot
- [x] Put the PR on our Project board

### CODE REVIEW

File: bitbots_ros_control/src/pressure_converter.cpp
Comment (timonegk) [inline bitbots_ros_control/src/pressure_converter.cpp:163]: What if `scale_and_zero_average_` is negative? The cast would make it a very large integer and the comparison would not make sense. Please cast the left hand side to whatever `scale_and_zero_average_` is (probably `int`).

**Review by timonegk — CHANGES_REQUESTED** [review]

**Review by timonegk — APPROVED** [review]

**Comment by jaagut** [comment]
(Don't merge, until marc has approved this)

**Review by SammyRamone — APPROVED** [review]

**Pull request merged**
